src/sass/components/_header.scss

```diff
@@ -26,15 +27,43 @@ header {
 
     .text-left ul li {
       margin-right: 1.5rem;
-
       @media (max-width: 767px) {
         margin-right: 0.4rem;
       }
-
       i {
         margin-left: 5px;
       }
     }
+    .text-left ul li ul li a {
+      color: #000;
+      margin-top: 5px;
+      padding: 0 5px;
+      height: 50px;
+      display: flex;
+      &:hover {
+        color: #fff;
+      }
+    }
+    .text-left ul li ul li {
+      display: none;
+    }
+    .text-left ul li ul {
+      position: absolute;
+      background-color: #fff;
+      box-shadow: 0 0 5px 0 #000;
+    }
+    .text-left ul li:hover ul li {
+      display: block;
+      height: 25px;
+      line-height: 3px;
+      padding: 7px;
+      width: auto;
+    }
+    .text-left ul li ul li:hover {
+      background-color: $primary;
+      width: 100%;
+      color: #fff;
```

Comment:
Ten atrybut color dałbym w osobnym selektorze .text-left ul li ul li:hover a, żeby kolor tekstu zmieniał się po najechaniu na li, a nie tylko po najechaniu na a.

Reply:
Podepnę tutaj jeszcze jedną uwagę, która jest dość podobna - obecnie jak najedziemy na element li w top-barze, to pokazuje nam się zagłębione menu, ale kolor tekstu zmienia się na żółty tylko po bezpośrednim najechaniu na a - moim zdaniem lepszy efekt uzyskamy, gdy kolor tekstu w top-barze będzie zmieniał się po najechaniu na cały li oraz gdy użytkownik będzie miał miał kursor na zagłębionym menu. :)

Reply:
Obie uwagi zostały zrealizowane.